allure-report-face/src/styles.css

```diff
@@ -56,6 +60,7 @@ html, body {
 }
 
 .long-line {
+  line-height: 1.5em;
```

Comment:
I don't agree.

Most usage examples are like this: https://github.com/allure-framework/allure-core/blob/91e05c3a2e467432f18ce1ed960e8dfc843a8ddd/allure-report-face/src/plugins/environment/environment-widget/EnvironmentWidget.hbs#L10

This line height is about how to add more space between table rows and has nothing about handling long titles, which can be in some headers or description blocks, whatever.